File: python/jax/train/vgg16/train_fashion.py
from python.jax.models.vgg16 import run_experiment

if __name__ == "__main__":
    run_experiment(
        dataset_path="data/fashion_mnist_png",
        output_file_base="vgg16_fashion",
        checkpoint_path="checkpoints/vgg16_fashion_jax.flax",
        img_size=(32, 32),
        epochs=30,
        batch_size=128
    )

# PYTHONPATH=. python3 python/jax/train/vgg16/train_fashion.py

File: R/train/vgg/train_fashion.r
source("R/models/vgg16.r")
cat("Script avviato!\n")
run_experiment(
    dataset_path = "data/fashion_mnist_png",
    checkpoint_path = file.path(root, "R/checkpoints/vgg16_fashion_r.pt"),
    img_size = c(32, 32), # 32x32 anche per Fashion
    grayscale = TRUE, # convertirà a 3 canali
    test_split = "test",
    epochs = 30,
    batch_size = 128
)